rockcraft.yaml

```diff
@@ -9,7 +9,11 @@ build-base: ubuntu:22.04
 platforms:
   amd64:
 
-cmd: ["lego"]
+services:
```

Comment:
Related to your comment in the PR description.

The pebble services section is not needed. Especially in the context of charms where we will create our own pebble service. We can simply remove it here imo.

Reply:
Fixed in https://github.com/canonical/lego-rock/pull/10/commits/c588d2c71ae80d0f84e283be56dcd566422debfe